rest-service/manager_rest/config.py

```diff
@@ -182,6 +183,37 @@ def load_from_db(self):
         # disallow implicit loading
         self.can_load_from_db = False
 
+    def update_db(self, config_dict):
+        """Update a config value.
+
+            Settings which have is_editable set to False, can only be edited
+            when passing the force flag.
+            If a schema is specified for the given setting, the new value
+            must validate.
+            """
+        from datetime import datetime
+        from flask_security import current_user
+        from manager_rest.storage import get_storage_manager, models
+        from manager_rest.manager_exceptions import ConflictError
+
+        sm = get_storage_manager()
+        names = [key for key in config_dict if not config_dict[key] is None]
+
+        query = sm.list(models.Config, None, filters={'name': names})
+        for item in query:
+            if not item.is_editable:
+                raise ConflictError('{0} is not editable'.format(item.name))
+            if item.schema:
+                try:
+                    jsonschema.validate(config_dict[item.name], item.schema)
+                except jsonschema.ValidationError as e:
+                    raise ConflictError(e.args[0])
+            item.value = config_dict[item.name]
+            item.updated_at = datetime.now()
+            item.updated_by = current_user
+            sm.update(item)
```

Comment:
It runs update query for each one, I think it is better to use `orm.Session` like in the function `load_from_db` so we will have one query to get all the values and second query to update all of them.